CONTRIBUTING.rst

```diff
@@ -68,7 +68,7 @@ Installing the latest code
 
 In order to ensure that any code changes are reflected in your installation, navigate to your cloned Nilearn base directory and install using the following command::
 
-    pip install -e .
+    pip install -e .[dev]
```

Comment:
I think it's important that you explain here, at least briefly, the logic of the [dev].
At least people should know that they should look at  setup.cfg

Reply:
I will add comments to `setup.cfg` and also point to that file in each of the places you commented on here.